lib/src/component_declaration/annotations.dart

```diff
@@ -164,6 +164,11 @@ class PropsMixin implements TypedMap {
   const PropsMixin({this.keyNamespace});
 }
 
+/// Annotation used to signify that this class should not be generated as an ancestor
+class ExportGeneratedAccessors {
+  const ExportGeneratedAccessors();
```

Comment:
Is this actually new? Seems like maybe it's leftover from when we were considering re-implementing inheritance?

Reply:
yep. Should get tossed.